packages/datetime/src/dateRangeInput.tsx

```diff
@@ -64,18 +75,47 @@ export class DateRangeInput extends AbstractComponent<IDateRangeInputProps, IDat
         // allow custom props for each input group, but pass them in an order
         // that guarantees only some props are overridable.
         return (
-            <div className={Classes.CONTROL_GROUP}>
-                <InputGroup
-                    placeholder="Start date"
-                    {...this.props.startInputProps}
-                    inputRef={this.refHandlers.startInputRef}
-                />
-                <InputGroup
-                    placeholder="End date"
-                    {...this.props.endInputProps}
-                    inputRef={this.refHandlers.endInputRef}
-                />
-            </div>
+            <Popover
+                autoFocus={false}
+                content={<DateRangePicker maxDate={this.props.maxDate} minDate={this.props.minDate} />}
+                enforceFocus={false}
+                inline={true}
+                isOpen={this.state.isOpen}
+                onClose={this.handlePopoverClose}
+                position={Position.TOP_LEFT}
```

Comment:
I'll change this to `Position.BOTTOM_LEFT` once there's enough content beneath the `DateRangeInput` example for the page not to scroll when the popover opens.